### PR TITLE
docs(RangeSlider): updates documentation template

### PR DESCRIPTION
**Pull request by toncabral88**
## Description
https://jirasoftware.catho.com.br/browse/QTM-374

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Code review
- [ ] Documentation

### CODE REVIEW

File: stories/RangeSlider/RangeSlider.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+CustomTooltipText.args = {
+  tipFormatter: value => `R$ ${value}`,
+};
+
+export const ControlledExample = () => {
+  const [value, setValue] = useState(0);
+
+  const handleChange = updatedValue => {
+    setValue(updatedValue);
+  };
```
Comment (MarcosViniciusPC) [inline stories/RangeSlider/RangeSlider.stories.jsx:59]:
```suggestion
  const handleChange = (_ev, updatedValue) => {
    setValue(updatedValue);
  };
```

Comment (MarcosViniciusPC) [reply stories/RangeSlider/RangeSlider.stories.jsx:59]: The first argument(updatedValue) is a synthetic event. To get the returned value include a second parameter.

**Review by MarcosViniciusPC — CHANGES_REQUESTED** [review]
Controlled Example is having problems when the value is changed. Here's a code suggestion for correction

File: stories/RangeSlider/RangeSlider.stories.jsx (hunk 1 above)
Comment (MarcosViniciusPC) [reply stories/RangeSlider/RangeSlider.stories.jsx:59]: However, this fix seems to lose the slider behavior. There must be another way to keep the behavior

**Review by MarcosViniciusPC — APPROVED** [review]
[x] Code Review
[x] Documentation

**Review by rapahaeru — APPROVED** [review]

**Pull request merged**